### PR TITLE
[OTX] Add Instance-Segmentation model configs

### PR DESCRIPTION
**Pull request by harimkang**
## Summary
Add Instance-segmentation model template & sample & test codes

### This PR includes :
- Add Instance-Segmentation model templates (Resnet50-MaskRCNN, EfficientNetB2B-MaskRCNN)
- Add Instance-Segmentation Sample code
- Add Instance-Segmentation test code (Integration) - CLI
- Fix some wrong value in detection

### Test Codes
```
# I-Seg Sample Code
bash otx/algorithms/detection/init_venv.sh ./venv python3.8
source venv/bin/activate
python -m otx.algorithms.detection.tools.instance_segmentation_sample otx/algorithms/detection/configs/instance_segmentation/resnet50_maskrcnn/template.yaml --export

# CLI Tests
pytest -s -v tests/integration/cli/detection/test_instance_segmentation.py
```

### CODE REVIEW

File: otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=invalid-name
+
+dataset_type = "CocoDataset"
+img_size = (1024, 1024)
+
+img_norm_cfg = dict(mean=(103.53, 116.28, 123.675), std=(1.0, 1.0, 1.0), to_rgb=False)
```
Comment (jaegukhyun) [inline otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py:23]: Is to_rgb=False on purpose? In detection all models pipeline's to_rgb is True

Comment (harimkang) [reply otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py:23]: This is the same setting as the latest develop.

**Review by jaegukhyun — COMMENTED** [review]
I leave a comment, pls check it

File: otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py (hunk 1 above)
Comment (jaegukhyun) [reply otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py:23]: Could you compare ote's mmdetection setting? If there is no special reason, to_rgb=True is appropriate

Comment (harimkang) [reply otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py:23]: This is the value used in all experiments so far, so it is better to leave it as it is.
This seems to require experimentation for comparing. Let's do it later.

Comment (jaegukhyun) [reply otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py:23]: Yeah sure, how about adding TODO flag?

Comment (harimkang) [reply otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py:23]: Thanks for the nice comment. i will add a flag :)

Comment (harimkang) [reply otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py:23]: Added TODO flag.

**Review by jaegukhyun — APPROVED** [review]
LGTM

**Pull request merged**